index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <script src="scripts/main.js" defer></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/mainStyle.css">
+    <title>Stephen's portfolio</title>
+</head>
+
+<body>
+
+    <header>
+        <nav class="toolbar">
+            <ul>
+                <li class="textPlaceholder"> <a href="#" aria-label="Ssenkooto Stephen"> Ssenkooto Stephen </a> </li>
+                <li> <button id="menuButton" type="button"> <img src="icons/menu.svg" alt=""> </button></li>
+                <li id="navigation">
+                    <ul class="container">
+                        <li class="textPlaceholder"><a href="#projects" aria-label="Portfolio"> Portfolio</a></li>
+                        <li class="textPlaceholder"><a href="#about" aria-label="About"> About</a></li>
+                        <li class="textPlaceholder"><a href="#contact" aria-label="Contact"> Contact</a></li>
+                        <li class="textPlaceholder"><a rel="noopener noreferrer" href="mailto:ssenkootos74@gmail.com"
+                                aria-label="Send Email"><img src="icons/mail.svg" alt="Send Email"></a></li>
+                    </ul>
+                </li>
+            </ul>
+        </nav>
```

Comment:
- [ ] As the previous reviewer mentioned, please kindly follow the given structure for the header:

```HTML
<header> 
    <a></a> 
    <nav> // hamburger icon goes here </nav>
</header>
```
Your implementation _mostly_ matches it, but it is important that **_the hamburger icon is enclosed with a `nav` tag more directly_**, as shown in the example above. Please kindly make this change. You can even preserve your current ancestor `nav` tag.

In the mobile view, the whole navigation is going to be in that button, implementing this will make your code more semantic and understandable for the current browsers 👍